src/docker/compose.dss-local.yaml

```diff
@@ -44,6 +47,21 @@ services:
     volumes:
       - "./processor/config.yaml:/config.yaml"
 
+  ws:
+    build:
+      context: ../../
+      dockerfile: src/docker/api/Dockerfile.ws
+      args:
+        - POSTGRES_WEBSOCKETS_VERSION=0.11.1.0
+    depends_on:
+      - diesel
+      - postgres
+    environment:
```

Comment:
Reading this, it looks like we may want an additional environment variable:
https://github.com/diogob/postgres-websockets#recovering-from-listener-database-connection-failures

> Self healing connection - postgres-websockets comes with a connection supervisor baked in. You just need to set the configuration PGWS_CHECK_LISTENER_INTERVAL to a number of milliseconds that will be the maximum amount of time losing messages. There is a cost for this since at each interval an additional SELECT query will be issued to ensure the listener connection is still active. If the connection is not found, the connection thread will be killed and respawned. This method has the advantage of keeping all channels and websocket connections alive while the database connection is severed (although messages will be lost).

Reply:
I think they're talking about the `LISTEN` they're using internally to listen for notifications. I added the environment variable with 1000ms as a value. I think it's a reasonable value that has a great balance between not spamming the DB and not having too much downtime.